avs_core/Files.cmake

```diff
@@ -16,6 +16,8 @@ FILE(GLOB AvsCore_Sources RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
 
   "core/os/PluginManager_win32.cpp"
   "core/os/PluginManager.h"
+  "core/os/*.cpp"
+  "core/os/*.h"
 
```

Comment:
Since os/ is for storing os-specific files, they shouldn't be globbed.  Otherwise we'll potentially get files for Linux being compiled on Windows or vice-versa.  That I moved PluginManager there is a bit of an artifact of when I tried bringing AvxSynth's plugin manager over, but I ultimately couldn't make that work, so I backtracked on it.  At this point we could just go ahead and revert the move, then there'd be no reason to keep core/os/ around.

Reply:
Well, pluginmanager now works for both linux and windows, so it no longer needs to be in /os. Findfirst might not work in MSVC and is independent of AviSynth, so it could stay there (although compilation conflicts can be also avoided with `#idfef AVS_LINUX`).

I don't know if GitHub allows you to modify this pull request. Feel free to add any changes you want.